lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -6702,7 +6708,38 @@ namespace Js
         Assert(element1 != NULL);
         Assert(element2 != NULL);
 
-        return JavascriptString::strcmp(element1->StringValue, element2->StringValue);
+        if (CONFIG_FLAG(StrongArraySort))
```

Comment:
Shouldn't this be `!CONFIG_FLAG(StrongArraySort)` ?

Reply:
Indeed, fixed.